### PR TITLE
[ci skip]Chinese annotation

### PR DESCRIPTION
**Pull request by VisualSJ**
CCProtocols.h
ccMacros.h

### CODE REVIEW

File: cocos/base/CCProtocols.h [hunk 1]
```diff
@@ -44,56 +47,90 @@ class CC_DLL __RGBAProtocol
...
      *
-     * @return  The opacity of sprite, from 0 ~ 255
+     * @~chinese 
+     * 返回不透明度。
+     * 
+     * 不透明度,表示节点透明或不透明。
+     * 0表示完全透明和255是完全不透明的。
+     * 
+     * @return  @~english The opacity of sprite, from 0 ~ 255
+     * @~chinese 精灵的不妥名都，范围从0 ~ 255
```
Comment (natural-law) [inline cocos/base/CCProtocols.h]: 错别字

**Comment by VisualSJ** [comment]
@natural-law  please review it again...Thanks

**Comment by natural-law** [comment]
@andyque 
This PR is ok for merge.

**Comment by zilongshanren** [comment]
@natural-law  Thanks.

**Pull request merged**